docs/superpowers/specs/2026-08-14-reverse-path-orchestrator-design.md

```
@@ -0,0 +1,259 @@
+
+What is missing is the **orchestrator** corinth-canal uses to close the loop:
+
+```text
+// corinth-canal src/model/core.rs — Model::forward_activity
+spike_train + potentials + iz
+  → projector.project(...) → embedding
+  → router.forward(&embedding) → expert_weights, selected_experts
+  → ModelOutput { embedding, expert_weights: Some(...), selected_experts: Some(...), ... }
+```
+
+#23 acceptance is an integration path with MoE fields asserted, without checkpoints or SAAQ. Extraction goal: hybrid-fusion owns this orchestration so corinth-canal can become thinner.
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)